package.json

```diff
@@ -105,12 +108,15 @@
     "stylelint": "^13.6.1",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "typescript": "^3.9.7",
+    "vue": "^2.6.12",
     "vue-docgen-api": "^4.30.0",
     "vue-jest": "^3.0.6",
     "vue-router": "^3.4.2",
+    "vue-template-compiler": "^2.6.12",
```

Comment:
Had an error the `vue@2.6.11` wasn't compatible with something that expected `2.6.12`. Updated it, then it complained that `vue-template-compiler` also need `2.6.12`